src/Symfony/Component/Validator/Constraints/Expression.php

```diff
@@ -54,12 +54,4 @@ public function getTargets()
     {
         return array(self::CLASS_CONSTRAINT, self::PROPERTY_CONSTRAINT);
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function validatedBy()
-    {
-        return 'validator.expression';
```

Comment:
yes, even if you change FrameworkBundle, it still breaks BC in the component.
and it also breaks BC for people registering their own service for this alias, because they won't have the new alias there. anything requiring a change elsewhere to keep it working is a BC break (this is the definition of a BC break: the old code does not work anymore)